build/ci/evergreen.yml

```diff
@@ -1145,7 +1145,7 @@ tasks:
       - func: "set-up ops manager"
       - func: "e2e test"
         vars:
-          E2E_TAGS: iam,om50
```

Comment:
what would happen next year for ops manager 6.0? will ops manager 6.0 become `opsmanager`?

Reply:
yes, this was my idea. Then we would drop om44 and use om50

Reply:
why not use om50 now and save some refactor later? I don't see why using `opsmanager` to indicate latest, is not intuitive

Reply:
I honestly removed it only because of this comment
> could we drop om50 given this should cover eventually om60?

I will update `opsmanager` with `om50`.